sor/src/main/java/com/bazaarvoice/emodb/sor/core/PendingCompaction.java

```diff
@@ -23,12 +24,12 @@
     private final UUID _initialCutoffId;
     private final Delta _delta;
     private final Delta _startingDelta;
-    private final List<UUID> _keysToDelete;
-    private final List<UUID> _compactionKeysToDelete;
+    private final List<DeltaClusteringKey> _keysToDelete;
+    private final List<DeltaClusteringKey> _compactionKeysToDelete;
     private final List<Map.Entry<UUID, Delta>> _deltasToArchive;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_deltasToArchive" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)